packages/core/src/interfaces/charts.ts

```diff
@@ -227,20 +246,7 @@ export interface StackedBarChartOptions extends BarChartOptions {
 /**
  * options specific to scatter charts
  */
-export interface ScatterChartOptions extends AxisChartOptions {
-	/**
-	 * options for the points
-	 */
-	points?: {
```

Comment:
While having this on ScatterChartOptions served Lollipops which use the same options, did nothing for Area and Combo charts